opslevel/resource_opslevel_property_definition.go

```diff
@@ -39,7 +39,7 @@ func resourcePropertyDefinition() *schema.Resource {
 func resourcePropertyDefinitionCreate(d *schema.ResourceData, client *opslevel.Client) error {
 	input := opslevel.PropertyDefinitionInput{
 		Name:   d.Get("name").(string),
-		Schema: opslevel.JSONString(d.Get("schema").(string)),
+		Schema: opslevel.NewJSON(d.Get("schema").(string)),
```

Comment:
Calling `NewJSON` would cause the `jsonencoded()` input to be double encoded, resulting in an unusable `schema`, AFAIK

Reply:
we should definitely test this out but i'm pretty sure this is correct as is.  `NewJSON` calls `unmarshal` on the string input so its turning a "string" into a `map[string]any` but as type `opslevel.JSON`. Which is what we want here.

Reply:
@rocktavious and @davidbloss you are both right, I was mistaken here since I confused `PropertyDefinition.Schema` and `Property.Value`. 

[The patch for property definitions in terraform is correctly using `opslevel.NewJSON()`.](https://github.com/OpsLevel/terraform-provider-opslevel/blob/8698a4c943741ce9dd3691c001c0e962dff8ab8e/opslevel/resource_opslevel_property_definition.go#L55)